STEERING-COMMITTEE.md

```diff
@@ -60,28 +59,27 @@ committee currently recognizes this delegated authority for:
 
 KSC meets every two weeks, or as-needed. Meetings are held online.
 
-Given the private nature of many of these discussions (e.g. privacy, private
+Given the private nature of some of these discussions (e.g. privacy, private
 emails to the committee, code of conduct violations, escalations, disputes
-between members, security reports, etc.) meetings are held in private.
+between members, security reports, etc.) some meetings are held in private.
 
-Meeting notes are available to members of the knative-dev mailing list (link to
-be added).
+Meeting notes are available to members of the [knative-dev mailing
+list](https://groups.google.com/forum/#!forum/knative-dev). Public meetings will
+be recorded and the recordings made available publicly.
 
```

Comment:
We have two mailing list knative-dev and knative-users, wondering if the steering community should have more visibility to the knative-users mailing list

Reply:
I suspect that Paul means that the notes will be in the team drive, which would grant read access to knative-users as well.

Reply:
That's correct and what I had meant. I'll make it explicit.